spec/support/rails_template.rb

```diff
@@ -57,8 +65,14 @@
   ENV['BUNDLE_GEMFILE'] = "#{File.expand_path(ENV['BUNDLE_GEMFILE'])}"
 RUBY
 
+# Setup webpacker if necessary
+if webpacker_app
+  rake "webpacker:install"
```

Comment:
I think its just a straight swap or other changes necessary @deivid-rodriguez?
```suggestion
  rails_command "webpacker:install"
```